hybrid/compartment_examples/call_sentry/main.c

```diff
@@ -0,0 +1,88 @@
+/***
+ * This example showcases two particular actions to help compartmentalization:
+ *      - reducing the bounds of the PCC (shared.s:82)
+ *      - calling a sentry from within a bounded compartment (shared.s:114)
+ *
+ * One difficulty, and capability leak, is how to return from the sentry
+ * function back to the caller with correct capabilities (i.e., PCC). In this
+ * example, we use `clr`, and manually return to it (main.c:49). However,
+ * having the `clr` available to the sentry function means it could access
+ * capabilities it should not be privy to (e.g., if the bounds of the sentry
+ * are set to only cover the specific function. In this example, this is
```

Comment:
Missing ")" before "."?

Reply:
Addressed in c9a02f2c81cc1b627dd38f71d2ab44bd3113fa16.